components/logicbutton/stories/logicbutton.stories.js

```diff
@@ -35,22 +38,43 @@ export default {
 		isDisabled: false,
 	},
 	parameters: {
-		componentVersion: version,
+		componentVersion: version,	
+		docs: {
+			story: {
+				height: "auto",
+			},
+		},
```

Comment:
```suggestion
		docs: {
			story: {
				height: "auto",
			},
		},
```

Can you leave this off of this PR - it breaks the S2 prep PR work I've done to improve container sizing in the docs.  Thanks!